lib/index.ts

```diff
@@ -0,0 +1,63 @@
+import * as _ from 'lodash';
+import { parse } from './indent-parser';
+
+export { parseLockFile };
+
+export interface PkgNode {
+  name: string;
+  version: string;
+  restriction: string;
+  // ignoring for now - addressing later
+  // depType?: DepType;
+  // targetFrameworks?: string[];
+}
+
+export interface PkgTree {
+  name: string;
+  dependencies: {
+    [dep: string]: PkgNode;
+  };
+  hasDevDependencies?: boolean;
+}
+
+// as with other dotnet-deps-parser we assume this parser will get the manifestFileContents as a string
+function parseLockFile(manifestFileContents: string) {
+  const depTree: PkgTree = {
```

Comment:
Please compare this to our depTree in Registry, that has the most up to date type